x-pack/plugins/security_solution/public/management/pages/event_filters/view/components/flyout/index.test.tsx

```diff
@@ -0,0 +1,169 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import React from 'react';
+import { EventFiltersFlyout } from '.';
+import * as reactTestingLibrary from '@testing-library/react';
+import { fireEvent } from '@testing-library/dom';
+import {
+  AppContextTestRender,
+  createAppRootMockRenderer,
+} from '../../../../../../common/mock/endpoint';
+import { MiddlewareActionSpyHelper } from '../../../../../../common/store/test_utils';
+
+import {
+  CreateExceptionListItemSchema,
+  ExceptionListItemSchema,
+} from '../../../../../../shared_imports';
+
+jest.mock('../form');
+jest.mock('../../hooks', () => {
+  const originalModule = jest.requireActual('../../hooks');
```

Comment:
Nice! first time I'm seeing the use of `jest.requireActual` :)

Reply:
Super useful when you want to partly mock something